controller.js

```diff
@@ -0,0 +1,10 @@
+const {selectCat} = require('./model.js')
+
+exports.getCat = (req, res, next) => {
+    selectCat() .then((categories) => {
+        res.status(200).send({categories})
+    })
+    .catch(err => {
+        next(err)
+    })
```

Comment:
This is picky but I think getCat and selectCat aren't the best names as if you had no idea what data this repo worked with, you would think it was getting cats 😅